File: python/list/LC561.py
# 짝수개의 nums, n쌍의 숫자 (a,b)... 중 min(a,b)들의 합이 최대가 되는 경우의 최대합

class Solution:
    def arrayPairSum(self, nums: list[int]) -> int:
        nums.sort()
        answer = 0
        for i in range(0,len(nums),2):
            answer += nums[i]

        return answer

File: java/leetcode/LC561.java
package leetcode;

import java.util.Arrays;

public class LC561 {
	// 2n 크기의 nums, n쌍 (a,b)의 최소들의 합이 최대가 되는 상태의 합
	// 정렬 후 0,2,4..의 인덱스만 더한다
	class Solution {
		public int arrayPairSum(int[] nums) {
			Arrays.sort(nums);
			int sum = 0;
			for (int i = 0; i < nums.length; i += 2) {
				sum += nums[i];
			}
			return sum;
		}
	}
}
